src/modules/emotionHandler/emotionHandler.cpp

```diff
@@ -0,0 +1,140 @@
+/******************************************************************************
+ *                                                                            *
+ * Copyright (C) 2023 Fondazione Istituto Italiano di Tecnologia (IIT)        *
+ * All Rights Reserved.                                                       *
+ *                                                                            *
+ ******************************************************************************/
+
+#include <yarp/os/ResourceFinder.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Bottle.h>
+#include "emotionHandler.h"
+
+#include <opencv2/opencv.hpp>
```

Comment:
Try to include what you use, and avoid if possible using these general headers that include everything